sdk/communication/azure-communication-identity/tests.yml

```diff
@@ -8,21 +8,16 @@ extends:
       ServiceDirectory: communication
       MatrixReplace:
         - TestSamples=.*/true
+      UseFederatedAuth: true
       CloudConfig:
         Public:
+          ServiceConnection: azure-sdk-tests
+          SubscriptionConfigurationFilePaths:
+            - eng/common/TestResources/sub-config/AzurePublicMsft.json
           SubscriptionConfigurations:
-            - $(sub-config-azure-cloud-test-resources)
             - $(sub-config-communication-services-cloud-test-resources-common)
```

Comment:
Can we use the new secret we created just for CTE here as well and remove the other ones ?

Reply:
Keeping it consistent across SDKs (and choosing the older keyvault instread of CTE)